pipelines/pingcap/tiflow/latest/pull_cdc_integration_test.groovy

```diff
@@ -140,7 +140,7 @@ pipeline {
                     axis {
                         name 'TEST_GROUP'
                         values 'G00', 'G01', 'G02', 'G03', 'G04', 'G05', 'G06',  'G07', 'G08', 'G09', 'G10', 'G11', 'G12', 'G13', 
-                            'G14', 'G15', 'G16', 'G17', 'G18', 'G19', 'G20', 'G21', 'G22', 'G23', 'G24', 'others'
+                            'G14', 'G15', 'G16', 'G17', 'G18', 'G19', 'G20', 'G21', 'G22', 'G23', 'G24'
```

Comment:
```suggestion
                            'G14', 'G15', 'G16', 'G17', 'G18', 'G19', 'G20', 'G21', 'G22', 'G23', 'G24', 'G25'
```

Reply:
Not recommended to continue adding test groups to avoid excessive resource consumption.